_posts/2020-03-30-exploting-spatial-diversity-using-unetstack3.md

```diff
@@ -0,0 +1,98 @@
+
+If you have a second modem available on the ship, you deploy it from another part of the ship. Or maybe there is a gateway buoy deployed nearby with a modem, and you can connect to its modem. Either way, the distributed spatial diversity technique (`Unity`) magically uses the information from both modems to recover connectivity to the AUV! 
+
+### So how to use `Unity` in UnetStack 3.1?
+
+The `Unity` is implemented as a premium agent and is available with the latest UnetStack 3.1 release. It allows users to transparently implement spatial diversity with a set of COTS UnetStack-based underwater acoustic modems, each with only a single receiver.
+
+Configuring and using the `Unity` agent to exploit spatial diversity is easy with just two simple steps:
+
+1. Setup the receiver nodes to cooperate.
+2. Add the `Unity` agent on the main receiver.
+
+
+#### 1. Set up for receiver nodes to cooperate (an example): 
+
+To set up the group of receivers to cooperate over a short-range network, we use [`Wormhole`](https://unetstack.net/handbook/unet-handbook_preface.html)  agent provided in the latest release UnetStack 3.1.  Transmitter node makes a transmission that is heard at all the receiver nodes. However, none of the nodes are able to successfully recover the information received, as the communication link is noisy.  In order to share the received noisy signals among the receivers, we can connect the receiver nodes using a `Wormhole`. A connection over a UDP link over a WiFi network is established by adding just a few lines of code on the receiver nodes as shown below:
+
+```groovy
+container.add 'udp', new UdpLink()            (1)
+container.add 'wormhole', new WormHole()      (2)
+wormhole.dsp = 'udp'                          (3)
+```
+Line number (1) adds a [`UdpLink`](https://unetstack.net/handbook/unet-handbook_wired_and_over_the_air_links.html) agent that implements a link protocol over UDP/IP for use over wired/wireless IP networks. The next step is to add the `Wormhole` agent and make it utilize the infrastructure provided by the `UdpLink` agent to allow the agents residing on receivers nodes to transparently talk to each other! 
+
+Line number (2) adds a `Wormhole` agent which allows the [fjåge](https://fjage.readthedocs.io/en/latest/) messages to be sent between containers over a [Unet link](https://unetstack.net/handbook/unet-handbook_introduction.html). 
+
+Line number (3) tells the `Wormhole` agent to utilize the `UdpLink` as the Unet link to share information. For more details on the `Wormhole` agent, please refer to this [link](https://unetstack.net/handbook/unet-handbook_preface.html).
```

Comment:
Wormhole was already linked in this section. I don't think you need to add a statement to refer to it again here.